client/CMT.cpp

```diff
@@ -229,9 +235,20 @@ int main(int argc, char * argv[])
 	std::cout.flags(std::ios::unitbuf);
 	console = new CConsoleHandler();
 	*console->cb = processCommand;
-	console->start();
+	console->start();	
+
+	// agent name for different log files
+	std::string agent = "";
+	if (vm.count("agent")) {
+		agent = vm["agent"].as<std::string>();
+	}
+	std::string file = "VCMI_CLIENT_";
+	std::string ext = ".log";
+	std::string filename = file.append(agent);
+	filename = filename.append(ext);
+	//

```

Comment:
'//'  - What does it mean?! Without such comments, please.

Reply:
The whole log section can be moved to a method setupClientLog() or something